pkg/controller/servicebindingrequest/reconciler_test.go

```diff
@@ -400,6 +400,45 @@ func TestEmptyApplicationSelector(t *testing.T) {
 	require.Equal(t, 0, len(sbrOutput.Status.Applications))
 }
 
+// TestEmptyApplicationSelector tests that Status is successfully updated when Service is not found
+func TestServiceNotFound(t *testing.T) {
```

Comment:
Please adjust this to a test table with the following cases:

* Reconcile without a service, to validate whether the `BindingReady` condition is `False`; and
* Reconcile with a service, to validate whether `BindingReady` condition is `True`.

Please observe any other conditions that might be checked as well.